src/includes/classes/Plugin.php

```diff
@@ -338,7 +348,10 @@ public function setup()
             'auto_cache_delay'       => '500', // In milliseconds.
             'auto_cache_sitemap_url' => 'sitemap.xml', // Relative to `site_url()`.
             'auto_cache_other_urls'  => '', // A line-delimited list of any other URLs.
-            'auto_cache_user_agent'  => 'WordPress',
+            'auto_cache_permalinks'  => '1', // `0|1`; auto-cache known Post/Page permalinks?
```

Comment:
I haven't figured this out yet, sorry.

My thinking is that I will try pull together a list of known permalinks and archives based on what is in the DB. That way the ACE will be capable of pre-caching a site for the most part, even when it lacks an XML sitemap.

This is a part of trying to accomplish a mode whereby ZenCache options could be preconfigured in such a way to create a sonic boom with very little effort on the part of a site owner. Those two options may or may not become a part of the UI directly, but having them in there might make it possible for different modes to be preconfigured in a future release.